components/usb/esp_tinyuf2/esp_tinyuf2.h

```diff
@@ -118,7 +118,7 @@ void esp_restart_from_tinyuf2(void);
  * @return
  *      - ESP_OK: Operation was successful.
  */
-esp_err_t esp_tinyuf2_set_all_key_hidden(bool if_hidden)
```

Comment:
@wormyrocks, is there any difference? Whether there is a formal parameter or not, makes no difference to the function itself.

Reply:
On my machine it won't compile without this change - something to do with the compiler settings in tinyuf2.
Outside of the include guards, all the other declarations in that file follow the same style.